content/en/cd-as-a-service/tasks/networking/install-agent.md

```diff
@@ -1,31 +1,103 @@
-* You have created client credentials. See the {{< linkWithTitle "cd-as-a-service/tasks/iam/client-creds.md" >}} guide for instructions.
-* You have access to a Kubernetes cluster and have installed [kubectl](https://kubernetes.io/docs/tasks/tools/).
-* You have installed [Helm](https://helm.sh/)(v3), which is used to install the Remote Network Agent.
+* You have a role that allows you to create **Client Credentials** and connect a Remote Network Agent.
+* You have access to your own Kubernetes cluster.
+
+## Generate install script using a UI wizard
+
+>You do not need to create **Client Credentials** for these options. The UI does that for you.
+
+### Option 1
+
+1. In the CD-as-a-Service Console, navigate to the **Welcome to Continuous Deployment-as-a-Service** [Configuration page](https://console.cloud.armory.io/configuration).
+1. Click **Connect your Kubernetes Cluster**.
+1. In the **Select Installation Method** window, select either **Connect Cluster Using Helm** or **Connect Cluster Using Kubectl**.
+1. In the **Identify Your Cluster** window, enter an agent identifier for your Remote Network Agent (RNA) in the **Cluster Name** field. You install this RNA in the cluster where you want to deploy your app, so create a meaningful identifier.
+1. Click **Continue**.
+1. Copy the script from the **Connect New Remote Network Agent** window and run it locally.
+
+### Option 2
+
+1. In the CD-as-a-Service Console, navigate to the [Configuration page](https://console.cloud.armory.io/configuration).
+1. Access the **Networking** > **Agents** screen.
+1. Click **Add an Agent**.
+1. In the **Name New Remote Network Agent** window, enter a name for your Remote Network Agent (RNA) in **Agent Identifier**. You install this RNA in the cluster where you want to deploy your app, so create a meaningful name.
+1. Choose **I want to use my own cluster.** in the **Choose Cluster Type** window.
+1. Copy the script in the **Install a Remote Network Agent** window and run it locally using kubectl.
+
+## Install manually using the CLI
+
+>You do not need to create **Client Credentials** for this option. The CLI does that for you.
+
+1. Install the CLI if you haven't already.
+
+   {{< include "cdaas/install-cli.md" >}}
+
+1. Log in using the CLI.
+
+   ```shell
+   armory login
+   ``` 
+
+1. Install the RNA in your cluster.
+
+   ```shell
+   armory agent create
+   ```
+
+   You choose your cluster and provide an **agent identifier** (cluster name) for the RNA during the installation process.
+
+## Install manually using kubectl
+
+### {{% heading "prereq" %}}
+
+* You have **Client Credentials**. See the {{< linkWithTitle "cd-as-a-service/tasks/iam/client-creds.md" >}} guide for instructions.
```

Comment:
I'd propose adding how to create credential steps right here, like we have CLI installation steps. It can be very high level

1. got this URL https://console.cloud.armory.io/configuration/credentials/new
2. Create a credential with "Remote Network Agent" permissions

Reply:
Added the reusable code from the Create Client Credentials page. Surrounded with `<details>` so user can expand for instructions.